src/components/ProfilePage/elements/FormControls.jsx

```diff
@@ -22,20 +24,19 @@ function FormControls({
           id={visibilityId}
           className="w-auto"
           type="select"
-          name="visibility"
+          name={visibilityId}
```

Comment:
Name should be unique now - before we were just sending 'visibility' up and relying on the formId up there to know which one it was.